mne/viz/topo.py

```diff
@@ -291,27 +505,25 @@ def plot_topo(evoked, layout=None, layout_scale=0.945, color=None,
     if not type(evoked) in (tuple, list):
         evoked = [evoked]
 
-    if type(color) in (tuple, list):
-        if len(color) != len(evoked):
-            raise ValueError('Lists of evoked objects and colors'
-                             ' must have the same length')
-    elif color is None:
-        colors = ['w'] + COLORS
-        stop = (slice(len(evoked)) if len(evoked) < len(colors)
-                else slice(len(colors)))
-        color = cycle(colors[stop])
-        if len(evoked) > len(colors):
-            warnings.warn('More evoked objects than colors available.'
-                          'You should pass a list of unique colors.')
-    else:
-        color = cycle([color])
+    if color is None:
+        color = ['#e41a1c', '#377eb8', '#4daf4a', '#984ea3', '#ff7f00',
+                 '#1b9e77', '#d95f02', '#7570b3', '#e7298a', '#66a61e']
+        color = color[:len(evoked)]
+    if 1 < len(color) < len(evoked):
+        warnings.warn('More evoked objects than colors available.'
+                      'You should pass a list of unique colors.')
+        color = cycle(color)
 
     times = evoked[0].times
+
     if not all([(e.times == times).all() for e in evoked]):
         raise ValueError('All evoked.times must be the same')
 
     info = evoked[0].info
     ch_names = evoked[0].ch_names
+    if plot_ch_names is None and len(ch_names) > 35:
```

Comment:
Completely arbitrary - I'm used to 32-channel layouts where showing channel names isn't a problem, and think if you go over that, it can become messy.